Rubberduck.CodeAnalysis/Inspections/Concrete/VariableRequiresSetAssignmentEvaluator.cs

```diff
@@ -41,6 +41,12 @@ public static bool RequiresSetAssignment(IdentifierReference reference, Rubberdu
                 return false;
             }
 
+            // For Each iterators are implicitly set.
+            if (reference.Context.GetAncestor<VBAParser.ForEachStmtContext>() != null)
```

Comment:
This will ignore all assignments within the block of a foreach Statement... I'm pretty sure that's not what was intended...

Reply:
You're correct - that was my bias in assuming that nobody would assign to the iterator inside the loop.